Update/onik_002.txt

```diff
@@ -3414,10 +3420,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(3, 2, "s19/02/990200154", 256, TRUE);
 	OutputLine(NULL, "「うん聞いて聞いて！",
-		   NULL, "\"Oh, yes~ so listen!", Line_WaitForInput);
+		   NULL, "\"Oh, yes~! So listen!", Line_WaitForInput);
```

Comment:
Leave tilde intact as a substitute for normal punctuation
Capitalizing "So" is good because these appears to be two distinct sentences.